### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Gallup93**
**Completed user stories 42,43,47,48**

Add test for enable/disable link on merchant items (story 42)

Refactor test to expect landing pages

Add merchant item enable/disable link, Add route to update item

Refactor items enable/disable route

Add update action for merchant items

Refactor test flash functionality

Refactor route/action name

Add test for edit link on merchant items from dashboard

Add merchant item edit links plus route/action

Add merchant item edit view page

Add description to items on merchant dash

Add merchant items edit spec (refactored from index spec), Add happy … …

Add test for merchant editing an item sad paths. (not finished), Fix … …

Refactor to expect description

Refactor item to not require image, Refactor tests to reflect changes

Add default image to items

Refactor item price to accept '$' in form submission, Refactor flash

Merge branch 'master' into mg_merchant_items

Remove nightmares

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -28,6 +27,11 @@ def update_inventory(quantity)
     update_attributes(inventory: (inventory - quantity))
   end
 
+  def default_image
+    self.image = "https://images.freeimages.com/images/large-previews/c72/banana-2-1330039.jpg"
```
Comment (muydanny) [inline app/models/item.rb:32]: bone

**Review by muydanny — APPROVED** [review]

File: spec/features/merchant/items/edit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        fill_in 'Price', with: "-3"
+
+        click_button "Update Item"
+      end
+      expect(current_path).to eq("/merchant/items/#{@shifter.id}/edit")
+      expect(page).to have_content("Price must be greater than -1")
+
+      within ".form" do
+        fill_in 'Price', with: "3"
+        fill_in 'Inventory', with: "-1"
```
Comment (stellakunzang) [inline spec/features/merchant/items/edit_spec.rb:94]: Super thorough test. LOVE TO SEE IT

**Review by stellakunzang — APPROVED** [review]
Great work Michael!

**Pull request merged**